docs/reference/firmware.md

```diff
@@ -8675,12 +8711,25 @@ Parameters:
 
   * format : format string
 
-`void operator()(const char *format, ...)`  
+`void log(const char *format, ...)`  
+`void operator()(const char *format, ...)`
+
+```cpp
+// EXAMPLE
+Log("This is %s message", "info");
```

Comment:
I would change the "info" parameter in case someone confuses it with actually setting the logging level. Something like

```
Log("The secret of everything is %d", 42);
```